_data/nvram.yaml

```diff
@@ -1,12 +1,29 @@
 ---
 - name: AAPL,ignore
+- platforms:
+  - macOS T2
+  name: ApChipID
+  scopes:
+  - 94b73556-2197-4702-82a8-3e1337dafbfb
```

Comment:
I'd say we should normalize like this, standard dash format lowercase which is what I often see in Apple products (EFI seems to use uppercase)

In theory, most UUID.parse methods will handle both gracefully, but caps hex I tend to reserve for "byte streams" (like a hex editor would show) when its a representation of a ID lowercase seems fine

This would probably apply to SHA hashes (canonical lower) or any other fixed width value

Thoughts?